src/appengine/handlers/cron/cleanup.py

```diff
@@ -839,10 +893,10 @@ def update_fuzz_blocker_label(testcase, issue,
       'to be found.' % (platform_message, fuzzer_name))
   if utils.is_oss_fuzz():
     update_message += OSS_FUZZ_INCORRECT_COMMENT
-  else:
+  elif utils.is_chromium():
     update_message += '\n\nMarking this bug as a blocker for next Beta release.'
-    update_message += INTERNAL_INCORRECT_COMMENT
-    update_message += (
+    update_message = _append_generic_incorrect_comment(
+        update_message, policy,
         ' and remove the %s label.' % data_types.ISSUE_RELEASEBLOCK_BETA_LABEL)
```

Comment:
should these have CHROMIUM_ISSUE_ prefix instead of just ISSUE_

Reply:
Done.